<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>398</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>398</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>442</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rzut ukośny</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="Graph" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>157</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Podaj V [m/s]:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="V_edit">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Podaj kąt [stopnie]:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="angle_edit">
             <property name="text">
              <string>45</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Podaj dt [sekundy]:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dt_edit">
             <property name="text">
              <string>0.001</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Skala X [metry]:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="X_scale">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>15</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Skala Y [metry]:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Y_scale">
             <property name="text">
              <string>15</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="Generate">
         <property name="text">
          <string>Generuj</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>151</width>
      <height>59</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Vx [m/s] =</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Vx_value">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Vy [m/s] =</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Vy_value">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>V [m/s]=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="V_value">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
